selfdrive/updated.py

```diff
@@ -319,7 +319,7 @@ def get_description(basedir: str) -> str:
       set_offroad_alert("Offroad_UpdateFailed", True, extra_text=extra_text)
     elif dt.days > DAYS_NO_CONNECTIVITY_MAX and failed_count > 1:
       set_offroad_alert("Offroad_ConnectivityNeeded", True)
-    elif dt.days > DAYS_NO_CONNECTIVITY_PROMPT:
+    elif dt.days > DAYS_NO_CONNECTIVITY_PROMPT and failed_count > 1:
```

Comment:
same thing here, easy to write again. move both of these after a new "else if failed_count > 1"